ethereum/remix-project
# fetch from etherscan while debugging

fix https://github.com/ethereum/remix-project/issues/2055

step to test it:
 - switch metamask to mainnet
 - switch remix to injected web3
 - open the debugger
 - input the transaction hash : 0x92867dc8f0b47ca8ca346283ff765d6f04d14ca0fa499a12a2a276fb6a7e3031
 - debugging should start with source highlighting
